pallets/automation-time/src/tests.rs

```diff
@@ -289,6 +293,33 @@ fn schedule_native_transfer_works() {
 	})
 }
 
+#[test]
```

Comment:
This happy path looks okay, and you need to create others to test its edge cases. For example,
1. A different schedule_fee asset should result in a different fee.
2. A different execution_fee asset should result in a different fee.
3.  A different schedule_as should result in a same fee.
4. A different flow should result in a different fee.

cc @v9n for a look.